VisionDemo/Podfile

```diff
@@ -0,0 +1,10 @@
+platform :ios, '14.0'
+
+target 'VisionDemo' do
```

Comment:
CocoaPods has been my dependency manager of choice mostly because of compatibility, but it's true that all frameworks I use are supported by SPM as well. The main difference is that CP includes all pods in a Pods project which is compiled every time I do a clean build.